Makefile

```diff
@@ -290,10 +290,11 @@ clean-docker: ## Deletes the docker containers for local development.
 
 govet: ## Runs govet against all packages.
 	@echo Running GOVET
-	$(GO) vet -shadow $(GOFLAGS) $(TE_PACKAGES) || exit 1
+	$(GO) get -u golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow
+	$(GO) vet -vettool=$(which shadow) $(GOFLAGS) $(TE_PACKAGES) || exit 1
```

Comment:
We need to run `$(GO) vet` in addition to an invocation with `-vetttool=$(which shadow)`, otherwise we'll only be running the shadow checker and not any of the default checks.